src/middlewares.js

```diff
@@ -23,6 +23,36 @@ const getMountForRequest = function (req) {
   return req.protocol + '://' + req.get('host') + mountPath;
 };
 
+const checkIPRange = (ip, ranges = []) => {
+  if (!ip) {
+    return false;
+  }
+  const allowAll = ['::', '::/0', '0.0.0.0/0'];
```

Comment:
I think this logic is something the dependency should provide, but it seems that `ipaddr.js` also doesn't do that? What is the benefit then of `ipaddr.js` over `ip-range-check`?

Reply:
Good point. `ipaddr.js` is used internally by `ip-range-check` and is a smaller, more popular library